common/activityLog/activityLog.js

```diff
@@ -99,6 +104,7 @@ export const getType = R.prop(keys.type)
 export const getContent = R.prop(keys.content)
 export const isSystem = R.propEq(keys.system, true)
 export const getDateCreated = ObjectUtils.getDateCreated
+export const isRecordDeleted = R.propEq(keys.recordDeleted, true)
```

Comment:
this would be record_uuid and it goes at line 111 (```// props associated from fetch```)

Reply:
ok